config/serverless.es.yml

```diff
@@ -12,6 +12,22 @@ xpack.serverless.observability.enabled: false
 enterpriseSearch.enabled: false
 xpack.fleet.enabled: false
 xpack.observabilityAIAssistant.enabled: false
+xpack.osquery.enabled: false
```

Comment:
__note:__ we agreed to disable `osquery` for Search project in Slack (cc @jimczi )

Reply:
++ thanks. adding @sphilipse and @linyaru for awareness.

Reply:
Seems like a reasonable choice to me